tests/test_utils.py

```diff
@@ -405,14 +405,8 @@ def test_convert_dict_to_env_variables(self):
             valid_env_items = convert_dict_to_env_variables(env)
         assert valid_env_items == ["ACCELERATE_DEBUG_MODE=1\n", "OTHER_ENV=2\n"]
 
-
-class TqdmTester(unittest.TestCase):
-    def test_tqdm(self):
-        expected = [0, 1, 2]
-        assert list(tqdm(range(3), main_process_only=True)) == expected
-        assert list(tqdm(main_process_only=True, iterable=range(3))) == expected
-
-        assert list(tqdm(range(3), main_process_only=False)) == expected
-        assert list(tqdm(main_process_only=False, iterable=range(3))) == expected
-
-        assert list(tqdm(range(3))) == expected
+    def test_tqdm_deprecation(self):
+        with pytest.warns(FutureWarning) as cm:
+            tqdm(True, range(3), disable=True)
+        assert "Passing `True` as the first argument to" in cm.pop().message.args[0]
```

Comment:
`pytest.warns` should take a `match` argument, which can be used instead of doing this assert.